test/functorch/test_vmap_registrations.py

```diff
@@ -30,6 +30,7 @@
     "aten::log_softmax.int",
     "aten::logdet",
     "aten::masked_select_backward",
+    "aten::max_pool1d",
```

Comment:
max_pool1d is CompositeImplicitAutograd. This means we should not actually be writing a batching rule for it, and we instead should be writing a batching rule for the things it decomposes into and instead put it into [BatchRulesDecomposition
](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/functorch/BatchRulesDecompositions.cpp)